src/java.desktop/share/classes/javax/swing/CellEditor.java

```diff
@@ -90,7 +90,7 @@ public interface CellEditor {
      * keep the selection from changing for some types of edits.
      * eg. A table that contains a column of check boxes, the user might
```

Comment:
```suggestion
     * keep the selection from changing for some types of edits,
     * eg. in a table that contains a column of check boxes, the user might
```
I think the sentence should be updated a bit, the part that stats with _“A table…”_ is not a new sentence, it shouldn't start with a capital letter, and without the preposition _“in a table”_ the sentence does not look good to me.


Two lines above, *“because is most cases”* should be changed to *“because **in** most cases”*.